libraries/botbuilder-core/src/configurationBotFrameworkAuthentication.ts

```diff
@@ -42,12 +41,12 @@ const TypedOptions = t.Record({
      *
      * Other values result in a custom authentication configuration derived from the values passed in on the [ConfigurationBotFrameworkAuthenticationOptions](xef:botbuilder-core.ConfigurationBotFrameworkAuthenticationOptions) instance.
      */
-    [AuthenticationConstants.ChannelService]: t.String.nullable().optional(),
+    [AuthenticationConstants.ChannelService]: t.Optional(t.Union(t.String, t.Null)),
```

Comment:
This is more clearly expresses the intent of the type and makes it more obvious what the resulting Typescript type will look like.